src/AElf.Launcher/appsettings.Docker.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ConnectionStrings": {
+    "BlockchainDb": "redis://redis:6379?db=1",
+    "StateDb": "redis://redis:6379?db=1"
```

Comment:
Can we try if "InMemory" works? See this -> https://github.com/AElfProject/AElf/blob/a5cc8ddd7509d4bc6d0a6f71cc4b253e186d85dc/src/AElf.Blockchains.MainChain/MainChainAElfModule.cs#L35-L39